source/gameengine/Ketsji/KX_PythonInit.cpp

```diff
@@ -1424,6 +1451,10 @@ static struct PyMethodDef rasterizer_methods[] = {
 	{"setAnisotropicFiltering", (PyCFunction)gPySetAnisotropicFiltering,
 	 METH_VARARGS, "set the anisotropic filtering level (must be one of 1, 2, 4, 8, 16)"},
 	{"getAnisotropicFiltering", (PyCFunction)gPyGetAnisotropicFiltering,
+	 METH_VARARGS, "get the anti aliasing level"},
+	{"setAntiAliasing", (PyCFunction)gPySetAntiAliasing,
+	 METH_VARARGS, "set the anti aliasing level (must be one of 1, 2, 4, 8, 16)"},
+	{"getAntiAliasing", (PyCFunction)gPyGetAntiAliasing,
```

Comment:
The changes are inserted in the middle of getAnisotropicFiltering braces {}, moreover the text points out to set but the function points out to get and viceversa.